mlrun/db/httpdb.py

```diff
@@ -274,6 +274,14 @@ def connect(self, secrets=None):
                 server_cfg.get("default_function_node_selector")
                 or config.default_function_node_selector
             )
+            config.default_function_priority_class = (
+                server_cfg.get("default_function_priority_class")
+                or config.default_function_priority_class
+            )
+            config.valid_function_priority_class_names = (
+                server_cfg.get("valid_function_priority_class_names")
+                or config.valid_function_priority_class_names
+            )
```

Comment:
See the comment
> These have a default value, therefore local config will always have a value, prioritize the API value first

These does **not** have a default value, the precedence can and should be the opposite